pyproject.toml

```diff
@@ -167,6 +167,11 @@ invoke = "~1.7.1"
 mkdocs-include-markdown-plugin = "~3.6.0"
 # Rendering docs to HTML
 mkdocs = "~1.3.0"
```

Comment:
Looks like these two dependencies should be bumped to ==3.6.1 and ==1.3.1 respectively.

Reply:
The `~` takes care of that, but since `poetry.lock` will influence what gets installed, I'm not sure where to stand on the exact versions vs this slightly flexible "any-later-patch" is OK.